README.md

```diff
@@ -407,6 +407,18 @@ These settings can be changed at any time.
   Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
 * `settings.ecs.enable-spot-instance-draining`: If the instance receives a spot termination notice, the agent will set the instance's state to `DRAINING`, so the workload can be moved gracefully before the instance is removed. Defaults to `false`.
 
+#### Image registry settings
+
+The following setting is optional and allows you to configure image registry mirrors and pull-through caches for your orchestrated containers.
+* `settings.image-registry.mirrors`: A mapping of image registry to a list of image registry URL endpoints.  When pulling an image from a registry, the container runtime will try the endpoints one by one and use the first working one.
+  * Example user data for setting up image registry mirrors:
+  ```
+  [settings.image-registry.mirrors]
+  "docker.io" = ["https://<my-docker-hub-mirror-host>", "https://registry-1.docker.io"]
+  "gcr.io" = ["https://<example-gcr-mirror>","https://<example-gcr-mirror-2>"]
```

Comment:
> Question, though - could we use https://gcr.io as the second entry in the gcr list, so it's a working entry, like the dockerhub URL in the line above?

Yes, we can. But it's also redundant (at least for containerd) since it would try the registry directly if all the mirrors fail. For docker it's not clear so I listed docker hub in the list as well as a fallback.

Reply:
Ah, ok, that's interesting.  If it's quick to confirm the difference, that'd be nice.  Maybe we could add a comment in the README above this?

Reply:
I just tried setting the mirror to some gibberish and ECS was still able to run a task on the node so I guess it fallsback by default. I'll go ahead and fix the example.

Reply:
It might be worth clarifying (with a comment in README) that it will still fall back to official sources - this isn't a way to insulate yourself from third parties.  That could be useful from the perspectives of debugging and security.